README.md

```diff
@@ -155,6 +155,14 @@ By default `fastify-cli` runs [`dotenv`](https://www.npmjs.com/package/dotenv),
 The default value for `--plugin-timeout` is 10 seconds.
 By default `--ignore-watch` flag is set to ignore `node_modules build dist .git bower_components logs .swp' files.
 
+#### Containerization
+
+When deploying to a Docker, and potentially other, containers, it is advisable to set a fastify address of 0.0.0.0 because these containers do not default to exposing mapped ports to localhost. 
```

Comment:
```suggestion
When deploying to a Docker, and potentially other, containers, it is advisable to set a fastify address of `0.0.0.0` because these containers do not default to exposing mapped ports to localhost. 
```

Reply:
No problem. Fixed.